pushnotifications/src/main/java/com/pusher/pushnotifications/internal/PushNotificationsInitProvider.kt

```diff
@@ -16,6 +19,14 @@ class PushNotificationsInitProvider: ContentProvider() {
 
       api.deviceId = deviceStateStore.deviceId
       api.setSubscriptions(deviceStateStore.interestsSet, OperationCallback.noop)
+
+      if (deviceStateStore.sdkVersion != BuildConfig.VERSION_NAME
```

Comment:
I'd add actual device data to this class too. You want manufacturer / model data here - because some custom roms will kill services in the background and it's data that we're interested in.

Reply:
Would something like [MODEL](https://developer.android.com/reference/android/os/Build.html#MODEL) and [MANUFACTURER](https://developer.android.com/reference/android/os/Build.html#MANUFACTURER) be enough or is there something else that we could use?

Reply:
I think this is currently outside the scope of the task.

We need the SDK version & OS version to send specific notifications payload. The model and manufacturer would just be analytics data, which we could collect at some point but are not needed now.